webui/react/src/pages/Dashboard.tsx

```diff
@@ -141,7 +142,7 @@ const Dashboard: React.FC = () => {
     });
 
   const emptyView = (
-    <div className={css.emptyMessage}>
+    <div className={emptyMessage.base}>
```

Comment:
suggestion: I can understand why you did this. Yeah kind of on the fence on this one too regarding whether it deserves it's own component or not. Looking at this PR and where it's used, I think it might make our lives easier if we do make this into its own component.

Reply:
hmm how about just sharing the css class? 
the css is the only real sharable part so far and if we add a component it'd just be an empty div that applies this CSS class.

Reply:
nit: I lean towards on the component implementation based on the reasons below, but I am ok if you want leave it as is, as this works too.
- This is our first pattern of sharing css instead of using a component. Though it's easy to trace where the styles are, I think components directory is the natural initial suspect.
- There's still a construction step of the empty view using a `div`. With a component what element gets used will just be predetermined.

Reply:
I separated empty message into a component.